cmd/photobak/main.go

```diff
@@ -38,6 +40,80 @@ func init() {
 	flag.BoolVar(&verbose, "v", verbose, "Write informational log messages to stdout")
 }
 
+type daemon struct {
+	repo       *photobak.Repository
+	repoMu     sync.Mutex
+	signalChan chan os.Signal
+}
+
+func startDaemon(interval time.Duration) {
+	d := daemon{signalChan: make(chan os.Signal, 1)}
+	signal.Notify(d.signalChan, os.Interrupt, syscall.SIGTERM)
```

Comment:
I didn't test whether it breaks the Windows build because I don't have any Windows machine, but I ran `cmd/photobak/build.bash` and it cross-compiles the binary for Windows without any error. Additionally as stated in https://msdn.microsoft.com/en-us/library/ms811896.aspx#ucmgch09_topic3 Windows should support SIGTERM, but I believe that Windows terminates processes using some other mechanism, so it won't work as expected but shouldn't break the application. If you wish, I'll wrap `signal.Notify(..., syscall.SIGTERM)` with `if runtime.GOOS != "windows"`.